.travis/mypy

```diff
@@ -16,36 +16,35 @@ tmp="$(mktemp -t mypy.XXXX)";
 #
 # * mypy notices trial's use of a skip marker attribute on functions and
 #   complains that it's not part of a function object's interface, so we
-#   ignore: has no attribute "skip"
+#   ignore: has no attribute "skip". Same with "todo".
 #
 # * Filters applied only to specific files are things to fix in either Klein or
 #   in mypy.
 #
+# * "type alias is invalid in runtime context" is a bug in mypy:
+#   https://github.com/python/mypy/issues/5354
+#
 # * We run mypy with disallow_untyped_defs=True to require that type
 #   annotations are used, but we ignore the errors for files that are not yet
 #   participating.
 #
 
 mypy "$@"      \
     | grep -v  \
-        -e ': error: "Callable\[\[[^]]*\], [^]]*\]" has no attribute "\(skip\|todo\)"'                            \
-        -e ': note: .* defined here'                                                                              \
-        -e '^src/klein/_app.py:[0-9:]*: error: All conditional function variants must have identical signatures'  \
-        -e '^src/klein/_app.py:[0-9:]*: error: Function is missing a type annotation'                             \
-        -e '^src/klein/_decorators.py:[0-9:]*: error: Function is missing a type annotation'                      \
-        -e '^src/klein/_iapp.py:[0-9:]*: error: Function is missing a type annotation'                            \
-        -e '^src/klein/_message.py:[0-9:]*: error: "attrib" does not return a value'                              \
-        -e '^src/klein/_plating.py:.* error: Function is missing a type annotation'                               \
-        -e '^src/klein/_plating.py:[0-9:]*: error: Need type annotation for '                                     \
-        -e '^src/klein/_resource.py:[0-9:]*: error: Function is missing a type annotation'                        \
-        -e '^src/klein/test/_trial.py:[0-9:]*: error: Function is missing a type annotation'                      \
-        -e '^src/klein/test/py3_test_resource.py:[0-9:]*: error: Function is missing a type annotation'           \
-        -e '^src/klein/test/test_app.py:[0-9:]*: error: Function is missing a type annotation'                    \
-        -e '^src/klein/test/test_plating.py:[0-9:]*: error: Function is missing a type annotation'                \
-        -e '^src/klein/test/test_plating.py:[0-9:]*: error: Need type annotation for '                            \
-        -e '^src/klein/test/test_resource.py:[0-9:]*: error: Function is missing a type annotation'               \
-        -e '^src/klein/test/util.py:[0-9:]*: error: Function is missing a type annotation'                        \
-        -e '^src/klein/test/util.py:[0-9:]*: error: Name .* already defined'                                      \
+        -e ': error: The type alias to [A-Za-z]* is invalid in runtime context'                          \
```

Comment:
This one is new.  Changes below are all removals.

Reply:
Some removals below are replaced by `type: ignore` comments in other files, to put that with the effected code, and because unused `type: ignore` comments are now flagged by `mypy`, so they are easier to clean up when they become obsolete.

Reply:
Removals are great!  And type: ignore is better than this shell junk (nominally Windows developers might want to work on Klein, and this is a silly reason to prevent them from doing so ...) so I'm glad to see this getting shorter